File: src/main/java/org/ozwillo/dcimporter/OzwilloDcImporterApplication.java
package org.ozwillo.dcimporter;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.scheduling.annotation.EnableScheduling;
import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;


@SpringBootApplication
@EnableScheduling
@EnableWebSecurity
@ComponentScan(basePackages = {"org.oasis_eu.spring","org.ozwillo.dcimporter"})

public class OzwilloDcImporterApplication {

	
	public static void main(String[] args){
		SpringApplication.run(OzwilloDcImporterApplication.class, args);
	
		}
}

File: src/main/kotlin/org/ozwillo/dcimporter/OzwilloDcImporterApplication.kt
package org.ozwillo.dcimporter

import org.springframework.boot.actuate.trace.http.HttpTraceRepository
import org.springframework.boot.actuate.trace.http.InMemoryHttpTraceRepository
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Bean
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
@EnableConfigurationProperties
@ConfigurationPropertiesScan(basePackages = ["org.ozwillo.dcimporter.config"])
class OzwilloDcImporterApplication {

    @Bean
    fun httpTraceRepository(): HttpTraceRepository = InMemoryHttpTraceRepository()
}

fun main(args: Array<String>) {
    runApplication<OzwilloDcImporterApplication>(*args)
}
